doc/README.md

```diff
@@ -59,12 +59,27 @@ cdk deploy
 >    * Create the event source (DynamoDB, S3 or Event Timer) for the Lambda functions
 >    * Add permissions for the Lambda to access Honeycode
 >    * Initialize the content in DynamoDB table, S3 bucket
-6. Once the deployment is complete, wait a couple of minutes and then verify the application is running correctly by using the following steps:
+
+7. Copy the **ContactHistoryBucketName** value that is printed on the terminal at the end of `cdk deploy`. Open the **s3-manifest.json** file in Cloud9 and update **BUCKETNAME** with the bucket name that you copied. Download this file to your machine as we will be using it to create the data set for visualization in Amazon QuickSight.
+
```

Comment:
Is there anyway possible to have CDK writing this file automatically? I wonder if something like this can be useful: https://github.com/aws/aws-cdk/issues/1773#ref-pullrequest-588250739

Reply:
Writing to local file in the manifest format would have required code outside CDK. Instead I have changed this to upload the manifest file to be uploaded to the contact history S3 bucket and the user can now use the URL to the file directly in QuickSight. This removes the steps of manually editing the s3-manifest.json file and downloading then uploading it to QuickSight

Reply:
I loved it!